python/ray/serve/tests/test_fastapi.py

```diff
@@ -151,6 +151,22 @@ def b(self, i: int):
     assert "get_current_servable" in str(self_dep.call)
 
 
+class Nested(BaseModel):
```

Comment:
Could you define and return these classes from a function? I'd still like to test that they're pickleable even when they're not importable.